pallets/committee/src/lib.rs

```diff
@@ -431,6 +439,36 @@ pub mod pallet {
 			Self::deposit_event(Event::NewConstituent(constituent));
 			Ok(())
 		}
+
+		/// Remove council or constituent via governance
+		///
+		/// This call can only be called after the approval of the committee
+		#[pallet::weight((T::WeightInfo::remove_member(), DispatchClass::Operational))]
+		pub fn remove_member(origin: OriginFor<T>, member: AccountIdFor<T>) -> DispatchResult {
+			T::ApprovedByCommitteeOrigin::ensure_origin(origin)?;
+
+			Members::<T>::try_mutate_exists(member.clone(), |m| -> Result<(), DispatchError> {
+				if let Some(ty) = m {
+					// Check if have enough council members
+					if *ty == MemberType::Council
+						&& <Members<T>>::iter_values().filter(|m| *m == MemberType::Council).count()
```

Comment:
let's stick to consistent turbofish syntax
```suggestion
						&& Members::<T>::iter_values().filter(|m| *m == MemberType::Council).count()
```